src/com/untamedears/realisticbiomes/GrowthConfig.java

```diff
@@ -261,103 +238,65 @@ public boolean isPersistent() {
 	}
 	
 	// given a block (a location), find the growth rate using these rules
+	
 	public double getRate(Block block) {
 		if (type == Type.FISHING_DROP) {
 			return getBaseRate(block);
 		}
-		// rate = baseRate * sunlightLevel * biome * (1.0 + soilBonus)
-		double rate = baseRate;
-		// if persistent, the growth rate is measured in growth/second
+		double rate= getBaseRate(block);
+		if (rate==0D) {
+			return rate;
+		}
 		if (isPersistent) {
 			rate = 1.0 / (persistentRate * SEC_PER_HOUR);
+			Double biomeMultiplier = biomeMultipliers.get(block.getBiome());
+			rate *= biomeMultiplier.floatValue();

```

Comment:
Dang, you're right. This is a bit weird then, if it's fish then we use base+biome, if not then we get base+biome, but if it's persisent we discard that and get persistence+(again)biome. Could this be rewritten to something more clearer like this:

		Double biomeMultiplier = biomeMultipliers.get(block.getBiome());
		if (biomeMultiplier == null || biomeMultiplier.floatValue() == 0.0D) {
			return 0.0;
		}
		
		if (type == Type.FISHING_DROP) {
			return baseRate * biomeMultiplier.floatValue();
		}
		
		double rate;
		if (isPersistent) {
			rate = 1.0 / (persistentRate * SEC_PER_HOUR);
		} else {
			rate = baseRate;
		}
		
		rate *= biomeMultiplier.floatValue();

and delete getBaseRate()?